frontend-react/src/utils/misc.ts

```diff
@@ -175,3 +175,27 @@ export const parseFileLocation = (
         fileName,
     };
 };
+
+export const removeHTMLFromString = (input: string) => {
+    return input.replace(/<(.|\n)*?>/g, "");
+};
+
+export const convertStringToSortableRepresentation = (
```

Comment:
Hmmm I'm not sure if this function is really useful on its own or if it should be built into the sorting function with type checks along the way.  One flag is that the content may not always be a string (as it's assumed `content` would be [here](https://github.com/CDCgov/prime-reportstream/pull/9085/files#diff-4a8691b6c15e1d775d398ac71c10833729efd0af84f55106b12ff9677a836f60R102)); it could be any type of primitive values -- most commonly `null`, a number, or a boolean --  all of which I think should be acceptable values in sorting, no?

Reply:
Happy to chat about this! I thought it would be nice to use this function to sort of show what inputs we "support" and that we could keep expanding the list. We could make an `enum` and make this more explicit.

Reply:
I'm going to move this out of `utils`, I agree it's too specific, especially with some of the other changes we've made.